packages/framework/src/Model/Product/ProductFacade.php

```diff
@@ -349,6 +342,8 @@ public function delete($productId)
         $this->em->remove($product);
         $this->em->flush();
 
+        $this->productSearchExportScheduler->scheduleProductIdForImmediateExport($productId);
```

Comment:
I do not understand what changed by this, but ok

Reply:
It’s suggested change to improve readability of a code. Prepare data, do something with data and then affect deleted product.